ansible_ai_connect/ai/api/exceptions.py

```diff
@@ -74,6 +74,10 @@ class WisdomAccessDenied(BaseWisdomAPIException):
     status_code = 403
 
 
+class WisdomNotFound(BaseWisdomAPIException):
+    status_code = 404
```

Comment:
To be this is more a 5xx error, since the request is valid, the end-point is correct however the backend cannot process the request.

Reply:
Careful what you wish for.